scapy/layers/dot11.py

```diff
@@ -247,6 +247,11 @@ def guess_payload_class(self, pay):
 
 class RadioTap(Packet):
     name = "RadioTap dummy"
+    deprecated_fields = {
+        "Channel": "ChannelFrequency",  # 2.4.3
```

Comment:
I wonder if the version should be part of the deprecation API. It might ease our future selves while debugging issues =)

Reply:
Good idea, will do